docs/EVM-compatible.md

```diff
@@ -9,51 +9,52 @@ The maximum EVM revision supported is `EVMC_BERLIN`.
 
 [pCKB](https://github.com/nervosnetwork/godwoken/blob/develop/docs/life_of_a_polyjuice_transaction.md#pckb) is a new concept introduced by Polyjuice.
 
-Recall that in Ethereum, the gas of each smart contract is calculated. The transaction fee is calculated then by multiplying gas with specified gas price. In Polyjuice, **pCKB** is used as the unit for calculating transaction fees. This means while the gas price in Ethereum is ETH/gas(which is denominated in wei, which is 1e-18 ether), in Polyjuice gas price is measured in pCKB/gas. When executing a transaction, Polyjuice will deduct transaction fee using the layer2 [sUDT](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0025-simple-udt/0025-simple-udt.md) type denoted by **pCKB**.
+Recall that in Ethereum, the gas of each smart contract is calculated. The transaction fee is calculated then by multiplying gas with specified gas price. In Polyjuice, **pCKB** is used as the unit for calculating transaction fees. This means while the gas price in Ethereum is ETH/gas(which is denominated in wei, which is 1e-18 ether), in Polyjuice gas price is measured in pCKB/gas. When executing a transaction, Polyjuice will deduct transaction fee using the layer-2 [sUDT](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0025-simple-udt/0025-simple-udt.md) type denoted by **pCKB**.
 
 Note when sending a transaction to a smart contract for certain behavior, the `value` of the transaction is `pCKB`.
 
-## All Tokens Are ERC20 Tokens
+## sUDT-ERC20 proxy contract
 
-Ethereum differs in the processing of ERC20 tokens, and native ETH tokens. This is also the reason why wETH is invented. Godwoken conceals this difference:
+When you use a sUDT token type, it will be represented in Godwoken as a layer-2 sUDT type. Polyjuice ensures that all the layer-2 sUDT tokens on Godwoken are in compliance with the ERC20 standard by the [sUDT-ERC20 Proxy Contract](../solidity/erc20/README.md). This contract provides a way for EVM code to interact with ERC20 standard interface to operate sUDT tokens on Godwoken as if they were ERC20 tokens.
 
-Whether you use a native CKB or any sUDT token type, they will all be represented in Godwoken as a layer2 sUDT type. Polyjuice starts from this layer2 sUDT [contract](https://github.com/nervosnetwork/godwoken-polyjuice/blob/b9c3ad4/solidity/erc20/SudtERC20Proxy_UserDefinedDecimals.sol) and ensures that all the tokens on Godwoken are in compliance with the ERC20 standard, no matter if they are backed by a native CKB or a sUDT. This means you don't need to distinguish between native token and ERC20 tokens. All you have to deal with is the same ERC20 interface for all different tokens.
+All you have to deal with is the same ERC20 interface for all bridged sUDT tokens.
```

Comment:
I don't quite understand this sentence. Would it be possible to clarify?

Reply:
Is it trying to say: “all the bridged sUDT tokens you are dealing with have the same ERC20 interface.” ？

Reply:
> Is it trying to say: “all the bridged sUDT tokens you will deal with have the same ERC20 interface.” ？

Yes. 
Do you have a better description?

Reply:
Then I think it could just be written simpler, removing "you have to deal with.." and directly saying: "In other words/That said/......, all bridged sUDT tokens will have identical ERC20 interfaces."

Reply:
> > Is it trying to say: “all the bridged sUDT tokens you will deal with have the same ERC20 interface.” ？
> 
> Yes. Do you have a better description?

Then I think it could just be written simpler, removing "you have to deal with.." and directly saying: "In other words/That said/......, all bridged sUDT tokens will have identical ERC20 interfaces."

Reply:
In other words, all bridged sUDT tokens have the same RRC20 interface thanks to the 1-to-1 sUDT-ERC20 proxy contract:

Reply:
> In other words, all bridged sUDT tokens have the same RRC20 interface thanks to the 1-to-1 sUDT-ERC20 proxy contract:

Typo RRC20

Reply:
https://github.com/nervosnetwork/godwoken-polyjuice/pull/178/commits/cf286358baee5d87a3c4a4112c75c0873cedb664